sections/event-types.txt

```diff
@@ -1572,15 +1572,9 @@ myDiv.addEventListener("auxclick", function(e) {
 			As with the EVENT{click} event type, the <a>default action</a> of
 			the EVENT{dblclick} event type varies based on the <a>event
 			target</a> of the event and the value of the {{MouseEvent/button}}
-			or {{MouseEvent/buttons}} attributes. Normally, the typical
+			or {{MouseEvent/buttons}} attributes. The typical
 			<a>default actions</a> of the EVENT{dblclick} event type match those
-			of the EVENT{click} event type, with the following additional
-			behavior:
-
-			*	If the <a>event target</a> is selectable, the <a>default
-				action</a> MUST be to select part or all of the selectable
-				content. Subsequent clicks MAY select additional selectable
-				portions of that content.
+			of the EVENT{click} event type.
```

Comment:
We also need to fix the last column for `dblclick` in Section 4.1 .

Reply:
Thanks for catching this, done!